### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14379,6 +14388,7 @@ throwgh->through
 thru->through
 thrue->through
 thruout->throughout
+thrusts->trusts
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: thrusts does exist!

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

**Comment by luzpaz** [comment]
@Gelma the git history for eefde9b will show `Add thrusts->trusts` can you remove it from the commit history ?

**Comment by Gelma** [comment]
> @Gelma the git history for [eefde9b](https://github.com/codespell-project/codespell/commit/eefde9bbc72730dcd0e2cdea4f41a25d3516955f) will show `Add thrusts->trusts` can you remove it from the commit history ?

Ok. Let's do this way: I close this PR, and re-open a new shiny clean.

**Comment by Gelma** [comment]
New one is this:
https://github.com/codespell-project/codespell/pull/1065

**Pull request closed without merging**